io.github.cortex_command_community.Cortex-Command-Community-Project.yml

```diff
@@ -0,0 +1,67 @@
+app-id: io.github.cortex_command_community.Cortex-Command-Community-Project
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: CortexCommand
+rename-desktop-file: CortexCommand.desktop
+rename-icon: io.github.cortex_command_community.Cortex-Command-Community-Project
```

Comment:
this is already the correct name....

Reply:
this also renames the icon name in the desktop file, otherwise it doesn't use a correct icon, will fix upstream

Reply:
then don't rename it at install time.